vignettes/git_usage.Rmd

```diff
@@ -43,23 +38,23 @@ Each feature branch must be related to an issue. We encourage new developers to
 
 ### Naming Branches
 
-The name of the branch must be prefixed with the issue number, followed by a short but meaningful description and the `@<target_branch>` suffix. The latter would be `@devel` in most cases. As an example, given an issue #94 "Program function to derive `LSTALVDT`", the branch name would be `94_derive_var_lstalvdt@devel`.
+The name of the branch must be prefixed with the issue number, followed by a short but meaningful description and the `@<target_branch>` suffix. The latter would be `@main` in most cases. As an example, given an issue #94 "Program function to derive `LSTALVDT`", the branch name would be `94_derive_var_lstalvdt@main`.
```

Comment:
`@main` does not need to be added as it is the fall back branch.

Reply:
removed the detail about main as it's not needed